EventFilter/CTPPSRawToDigi/plugins/CTPPSPixelRawToDigi.cc

```diff
@@ -35,8 +35,14 @@ CTPPSPixelRawToDigi::CTPPSPixelRawToDigi( const edm::ParameterSet& conf )
   produces< edm::DetSetVector<CTPPSPixelDigi> >();
 
 //CablingMap could have a label //Tav
-  mappingLabel_ = config_.getParameter<std::string> ("mappingLabel"); //RPix
+  includeErrors_ = config_.getParameter<bool> ("IncludeErrors");
+  mappingLabel_ = config_.getParameter<std::string> ("mappingLabel"); 
 
+  if(includeErrors_){
+    produces< edm::DetSetVector<CTPPSPixelDataError> >();
+    produces<DetIdCollection>();
+    produces<DetIdCollection>("UserErrorModules");
```

Comment:
it looks like only the DetSetVector product is actually produced (written in the event).
Please either remove mention of the two DetIdCollection if they will not be filled in a relatively near future or actually put them in the event (even if empty for now).